src/test/java/com/google/sps/servlets/LiveCourseDataTest.java

```diff
@@ -89,14 +89,15 @@ public void FindingExisitngTermEntity() {
   public void GettingRatingData_Hours_Difficulty() {
     DatastoreService db = DatastoreServiceFactory.getDatastoreService();
     Entity parent = addTermEntity(db);
-    addRatingEntity(db, 12, 7, parent.getKey());
+    addRatingEntity(db, /* hours */ 12, /* difficulty */ 7, parent.getKey());
     addRatingEntity(db, 7, 4, parent.getKey());
```

Comment:
Nit: would be nice to add the hours/difficulty annotations here too

Reply:
Just wondering for the future. If I have a case where I am adding like 5 entities would i need to add a comment for each one?

Reply:
It should be just copy/paste yes? We only do it when it's not obvious what you're passing (so raw primitives basically, ints, longs, strings).